content/en/platform/corda/4.8/open-source/key-concepts-transactions.md

```diff
@@ -20,96 +20,89 @@ title: Transactions
 
-* Notary-change transactions (used to change a state’s notary - see [Notaries](key-concepts-notaries.md))
-* General transactions (used for everything else)
+* Notary-change transactions, to change a state’s [notary](key-concepts-notaries.md).
+* General transactions, for everything else.
 
-## Transaction chains
+## Transaction backchains
 
-When creating a new transaction, the output states that the transaction proposes do not exist yet, and must
-therefore be created by the proposer or proposers of the transaction. However, the input states already exist as the outputs of
-previous transactions. We therefore include them in the proposed transaction by reference.
+Transaction backchains let a [node](key-concepts-node.md) verify that each input was generated from a valid series of transactions. This is called "walking the chain." If you need to break this chain (for example, because you want to increase performance by reducing the number of transactions the node has to check, or because you want to keep previous transactions private) you can [reissue states](reissuing-states.md).
 
-These input states references are a combination of:
+Backchains are created as *input state references* link together over time. Input state references let you use the outputs of previous transactions as the inputs of new transactions.
 
-* The hash of the transaction that created the input
-* The input’s index in the outputs of the previous transaction
+Input state references consist of:
 
-This situation can be illustrated as follows:
+* The [hash](https://www.investopedia.com/terms/h/hash.asp) of the transaction that created the input.
+* The input’s index (location in the backchain) in the outputs of the previous transaction.
+
+You can see how this works in this example transaction:
 
 {{< figure alt="tx chain" width=80% zoom="/en/images/tx-chain.png" >}}
-These input state references link transactions together over time, forming what is known as a *transaction chain*.
 
-{{< note >}}
-See [Reissuing states](reissuing-states.md) for information about reissuing states with a guaranteed state replacement, which allows you to break transaction backchains.
-{{< /note >}}
 
-## Committing transactions
+## Committing transactions to the ledger
 
-Initially, a transaction is just a **proposal** to update the ledger. It represents the future state of the ledger
-that is desired by the transaction builders:
+Initially, a transaction is only a proposal to update the ledger. It represents the future state of the ledger desired by the transaction builders.
 
 {{< figure alt="uncommitted tx" width=80% zoom="/en/images/uncommitted_tx.png" >}}
-To become reality, the transaction must receive signatures from all of the *required signers* (see **Commands**, below). Each
-required signer appends their signature to the transaction to indicate that they approve the proposal:
+To become reality, the transaction must receive signatures from all the *required signers*. Each
```

Comment:
This is a bit tricky - the transaction is a proposal to update the ledger in the first para, but here we are saying there is a further step which makes it a reality. Do we need to change what we say a transaction is? the 'proposal to update the ledger' definition may need to be rethought.

Reply:
Fuzzy wording there. Updated "become reality" to "committed to the ledger"